### ISSUE TITLE
[Question] Multiple subnet in HOME_NET

### ISSUE THREAD

**Issue by WingerHusar** [snort3/snort3#179]
Hi, 
I have one question:
How Can I define multiple subnet in snort 3.1.4.0 ?
Currently it looks like:

HOME_NET = "[10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/24]"
EXTERNAL_NET = "!$HOME_NET"

Some Rules like below:

alert ip $EXTERNAL_NET any -> $HOME_NET any

are triggering even from network 192.168.X.X to 172.16.X.X and I don't know why. 

Is shouldn't be triggering only from public addresses ?

**Comment by allewi** [comment]
Yes.. that looks correct. If the syntax was incorrect there should be an error at startup. Can you provide an example of some traffic that shouldnt be alerting but is? 

Also.. these type of questions are best suited for the snort-users email list.
